website/server/controllers/api-v3/auth.js

```diff
@@ -574,11 +609,11 @@ api.deleteSocial = {
   async handler (req, res) {
     let user = res.locals.user;
     let network = req.params.network;
-
-    if (network !== 'facebook') throw new NotAuthorized(res.t('onlyFbSupported'));
-    if (!user.auth.local.username) throw new NotAuthorized(res.t('cantDetachFb'));
-
-    await User.update({_id: user._id}, {$unset: {'auth.facebook': 1}}).exec();
+    if (supportedSocialNetworks.indexOf(network) === -1) throw new NotAuthorized(res.t('unsupportedNetwork'));
+    if (!user.auth.local.username) throw new NotAuthorized(res.t('cantDetachSocial'));
```

Comment:
So if you've attached via Google _and_ Facebook, you can't detach one of them if you don't also have a local?